packages/reports/reports.hjson

```diff
@@ -0,0 +1,212 @@
+              "Family_Planning_Acceptors_Natural_Method",
+              "Family_Planning_Acceptors_Depo_Provera",
+              "Family_Planning_Acceptors_Jadelle",
+              "Family_Planning_Acceptors_Pill_Single",
+              "Family_Planning_Acceptors_Sterilization_Male",
+              "Family_Planning_Acceptors_Sterilization_Female",
+              "Family_Planning_Acceptors_Pill_Combined",
+              "Family_Planning_Acceptors_Condom_Male",
+              "Family_Planning_Acceptors_IUD",
+              "POP21AND23",
+              "POP15",
+              "POP17AND19",
+              "POP13",
+              "POP11"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": false
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+            "transform": "select",
+            "$.dataElement": "$.value",
+            "'period'": "$.period"
+        },
+        {
+          "transform": "aggregate",
+          "period": "group",
+          "...": "sum"
+        },
+        // {
+        //   "transform": "select",
+        //   "'Contraceptive Prevalence Rate'": "100 * add([$.Family_Planning_Acceptors_Other, $.Family_Planning_Acceptors_Condom_Female, $.Family_Planning_Acceptors_Natural_Method, $.Family_Planning_Acceptors_Depo_Provera, $.Family_Planning_Acceptors_Jadelle, $.Family_Planning_Acceptors_Pill_Single, $.Family_Planning_Acceptors_Sterilization_Male, $.Family_Planning_Acceptors_Sterilization_Female, $.Family_Planning_Acceptors_Pill_Combined, $.Family_Planning_Acceptors_Condom_Male, $.Family_Planning_Acceptors_IUD]) / add([$.POP21AND23, $.POP15, $.POP17AND19, $.POP13, $.POP11])",
+        //   "'period'": "$.period"
+        // }
+      ],
+      "output": []
+    }
+  },
+  {
+    "id": "UNFPA_Region_Facilities_Offering_Services_At_Least_1_Family_Planning",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "RHS4UNFPA809",
+              "RHS1UNFPA03"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": true
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+            "transform": "select",
+            "$.dataElement": "$.value",
+            "'period'": "convertToPeriod($.period, 'QUARTER')",
+            "'organisationUnit'": "$.organisationUnit"
+        },
+        {
+            "transform": "filter",
+            "where": "not (exists($.RHS4UNFPA809) and gt(1, $.RHS4UNFPA809))"
+        },
+        {
+            "transform": "aggregate",
+            "organisationUnit": "group",
+            "period": "group",
+            "...": "last"
+        },
+        {
+            "transform": "select",
+            "'organisationUnit'": "'COUNTRY'",
```

Comment:
Is this doing something special? Or is it literally just setting `organisationUnit: 'COUNTRY'`?

Reply:
Nope, you're right it's just setting it to the string `'COUNTRY'`. What this should really be is something like:
`"'organisationUnit'": "getCountryCode($.organisationUnitCode)"`

But I haven't got anything like that implemented as of yet.